pkgs/servers/matrix-conduit/default.nix

```diff
@@ -1,22 +1,40 @@
-{ stdenv, lib, fetchFromGitLab, rustPlatform, }:
+{ stdenv, lib, fetchFromGitLab, rustPlatform, pkgs, ... }:
```

Comment:
```suggestion
{ stdenv, lib, fetchFromGitLab, rustPlatform, pkgs }:
```

Are the `…` needed for a specific reason?

Reply:
No reason. Fixed.